apps/src/netsim/DashboardUser.js

```diff
@@ -91,7 +91,9 @@ DashboardUser.getCurrentUser = function () {
         DashboardUser.currentUser_.initialize_(data);
       },
       error: function (/*jqXHR, textStatus, errorThrown*/) {
-        throw new Error("Unable to retrieve current user info.");
+        DashboardUser.currentUser_.initialize_({
```

Comment:
I don't know if I have a super strong opinion here. My concern with this is that looking at this line in isolation, I can distinguish it from the case where I'm truly calling private methods. i.e. if i see a call to <obj>.foo_() where obj is anything other than this or self, it looks like I'm accessing a private. You may be technically correct that it's ok here, but if this pattern is common, it becomes harder to pick out true private accessors.